src/lib/components/topbar/topbar.scss

```
@@ -5,10 +5,15 @@
   background-color: $action-lighten-5;
   box-sizing: border-box;
   display: flex;
-  justify-content: flex-end;
+  justify-content: space-between;
   padding: $space-16 $space-32;
   width: 100%;
 
+  &__topright-container {
+    display: flex;
```

Precisa do align-items: center pra ficar tudo centralizado horizontalmente.

```suggestion
    align-items: center;
    display: flex;
```